Make.defs

```diff
@@ -2,11 +2,18 @@ CC = gcc
 INSTALL = install -p
 MKDIR = mkdir
 
+# Flag to enable -Werror on build.
+WERROR =
+
 DBGFLAG = #-ggdb
 ifndef CFLAGS
 CFLAGS = -O2
 endif
-CFLAGS += $(DBGFLAG) -I../include -Wall
+CFLAGS += $(DBGFLAG) -I../include
+CFLAGS += -W -Wall -Wextra -Wpedantic
```

Comment:
At least in gcc `-Wextra` is the modern name of `-W`.
Is there any compiler that implements `-W` and `-Wextra` as different options?

Reply:
Let's drop `-W`then.

Reply:
`-Wextra` is kind of less portable than `-W`, but it wouldn't be an easy task to find a supported compiler that doesn't implement `-Wextra`.

Reply:
So using `-Wextra`should be fine?

Reply:
I'm OK with either of them.

Reply:
Let's use `-Werror`then, as it look more descriptive.